src/Container/PackageProxyContainer.php

```diff
@@ -58,7 +59,7 @@ public function has(string $id): bool
     /**
      * @return bool
      *
-     * @throws \Exception
+     * @throws ContainerException
```

Comment:
I think this change is not correct because the method doesn't throw `ContainerException`. 

Honestly, I'm not sure does the method throw an exception somewhen :smile: (this means the `has` method doesn't throw the exception as well).

Reply:
If this method does not throw any exception at all this declaration shall be removed. 
I was pretty sure the exception could be thrown from some nested calls.